drivers/usb/dwc2/platform.c

```diff
@@ -439,6 +439,9 @@ static int dwc2_driver_probe(struct platform_device *dev)
 	struct dwc2_hsotg *hsotg;
 	struct resource *res;
 	int retval;
```

Comment:
Let's add "#ifdef CONFIG_MACH_STM32H743" here to avoid extra code on other platforms